### PR TITLE
Create an interface to illustrate, make adjustments to align with HQCS and update the IPO comments.

### PR DESCRIPTION
**Pull request by phuocvo832004**
To validate user input and check if a number is prime, require:

1) Add UI: 
1.1 Page structure: Implemented HTML form for number input and submit button. 
1.2 Page style: Added CSS for layout, input styling, and result display with different colors based on the outcome.

2) Add input validation:
Validate user input to ensure it’s a number greater than 1.

3) Add logic to check primality:
Implemented JavaScript logic to determine if the input number is prime.

4) Add result output:
Display the result in a styled div element, changing text and color based on whether the number is prime or not.

Output:
Some images showing PR changes/output:
The initial form with a number input field and a submit button:
image
The result displayed when the input is a prime number:
image
The result displayed when the input is not a prime number:
image

### CODE REVIEW

File: demo.js [hunk 1]
```diff
@@ -1,11 +1,37 @@
...
-  };
\ No newline at end of file
+
+    let isPrime = true;
+    for (let i = 2; i <= Math.sqrt(number); i++) {
+        if (number % i === 0) {
+            isPrime = false;
+            break;
+        }
+    }
```
Comment (thangphanmoso) [inline demo.js]: CS5.2: em update thêm comment cho logic này nhe

File: index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+</head>
+<body>
+    <h1>Kiểm tra số nguyên tố</h1>
+    <form id="primeForm">
+        <!-- Input: Enter the number to check (HTML form) -->
+        <label for="number">Nhập số cần kiểm tra:</label>
+        <input type="number" id="number" name="number" required>
+        <button type="submit">Kiểm tra</button>
+    </form>
+    <div id="result"></div>
```
Comment (thangphanmoso) [inline index.html]: CS5.2: em update thêm comment cho các section/block này nhe
```
// .. some comment here
<h1>

<ngoài ra nên add thêm empty line ntn để tách biệt các section/block cho dễ đọc code>
// .. some comment here
<form>

// .. some comment here
<div>
```

File: style.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
+body {
+    font-family: Arial, sans-serif;
+    margin: 20px;
+    background-color: #f0f0f0;
+    color: #333;
+}
+h1 {
```
Comment (thangphanmoso) [inline style.css:10]: CS5.2: em update thêm empty line và bổ sung comments nhe

**Review by thangphanmoso — CHANGES_REQUESTED** [review]

**Review by phuocvo832004 — COMMENTED** [review]
I solved all comment issues.

**Review by thangphanmoso — APPROVED** [review]

**Pull request merged**